First-outlook.md

```diff
@@ -50,12 +50,11 @@ Sidenote (Windows): When using HashLink you can run them very quickly when [addi
 
 ## 📒 Discover Haxe
 
-- [Browse **Haxe libraries**](https://lib.haxe.org/) and benefit of the full potential of available Haxe libraries
-  - [understanding haxelib](https://lib.haxe.org/documentation/using-haxelib/)
-- Becoming proficient in **Haxe** will provide you with completely new ***abilities and powers*** in your code. So learning more about what Haxe has to offer as a language gives you new options to solve things. These are just some examples (for more [visit Haxe.org](https://haxe.org/)):
-  - the [Haxe manual](https://haxe.org/manual/)
-  - the [Haxe API](api.haxe.org/)
-  - the [Haxe wiki](https://github.com/HaxeFoundation/haxe/wiki) ( <- quite hidden!)
-  - the [Haxe code cookbook](https://code.haxe.org/)
-  - the [Haxe community](http://community.haxe.org/) (also visit haxe.org and see "Connect" options)
+Heaps is powered by Haxe. Learn more about Haxe here:
+  - The [Haxe website](https://haxe.org/):
+  - The [Haxe manual](https://haxe.org/manual/)
+  - The [Haxe API](api.haxe.org/)
```

Comment:
Might be good to fix that url while you're at it

Reply:
@kLabz Nice catch. Fixed!